src/test/java/uk/gov/dhsc/htbhf/steps/NavigationSteps.java

```diff
@@ -18,6 +18,12 @@ public void enterDetailsUpToPage(String page) {
         enterDetailsUpToPage(PageName.toPageName(page));
     }
 
+    @Given("^I am in (.*) page")
+    public void goToPage(String pageName) {
+        BasePage page = getPages().getPageByName(PageName.toPageName(pageName));
+        page.openDirect();
+    }
```

Comment:
This exact functionality already exists in this steps class, either use `I navigate to the (.*) page` in your feature file, or you can add that extra syntax to the existing method as such: 
`@When("^I navigate to the (.*) page|I am  in the (.*) page")`

Reply:
changed the step to I navigate to the page